packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -51,16 +51,23 @@ describe( 'Widgets screen', () => {
 	beforeAll( async () => {
 		// TODO: Ideally we can bundle our test theme directly in the repo.
 		await activateTheme( 'twentytwenty' );
-		await deactivatePlugin(
-			'gutenberg-test-plugin-disables-the-css-animations'
-		);
+		// Reduced motion is needed to immediately show and dismiss the snackbars.
+		await page.emulateMediaFeatures( [
+			{
+				name: 'prefers-reduced-motion',
```

Comment:
A long time ago, I tried removing our `gutenberg-test-plugin-disables-the-css-animation` plugin with this `page.emulateMediaFeatures`. I didn't manage to make it work at that time and it was not clear way. Maybe it's time to try that one more time :)

Reply:
I believe only some of the animations/transitions are respecting `prefers-reduced-motion`. For instance, the transition effect in the customizer doesn't (https://core.trac.wordpress.org/ticket/53542). Maybe we should enable them both?

Reply:
the combination of the `gutenberg-test-plugin-disables-the-css-animation` and the `FORCE_REDUCED_MOTION` env variable that is enabled in e2e test builds disable all "Gutenberg animations" (the JS and the CSS ones), I do believe that ideally all animations should respect `prefers-reduced-motion` (including core ones) which might help us get rid of all these hacks (plugin and env variable)

Reply:
`FORCE_REDUCED_MOTION` is only enabled on CI, and we have to manually rebuild the whole app if we're writing tests locally. Puppeteer's API seems like a better alternative here.

Yep, ideally, but we're not quite there yet 😅.